routes/image_routes.js

```diff
@@ -17,4 +17,15 @@ module.exports = function(router) {
       res.end();
     });
   });
+
+  router.get('/about', function (req, res) {
+    fs.readFile('./public/index.html', function (err, data) {
+      res.writeHead(200, {'Content-Type': 'text/html'});
+      if (err) {
+        return console.log(err);
+      }
+      res.write(data);
+      res.end();
+    });
+  })
```

Comment:
Our Grunt Tests will not like line 30 in your code. Please add a semi-colon. Also, line 21, remove the space between your function declaration and the req, res parameters. Should look like... function(req, res)....